src/web/src/components/AdvancedSearch.tsx

```diff
@@ -0,0 +1,62 @@
+import { Grid } from '@material-ui/core';
+import { createStyles, makeStyles } from '@material-ui/core/styles';
+import FormControl from '@material-ui/core/FormControl';
+import useSearchValue from '../hooks/use-search-value';
+
+interface Props {
+  openDialog: boolean;
+}
+
+const useStyles = makeStyles(() =>
+  createStyles({
+    root: {
+      margin: '20px',
+    },
+
+    input: {
+      width: '100%',
+      boxSizing: 'border-box',
+      borderRadius: '20px',
+      padding: '10px 15px',
+      outline: 'none',
+      border: 'solid 2px #121D59',
+    },
+    criteriaText: {
+      fontSize: '18px',
+      marginTop: '17px',
+    },
+  })
+);
+
+const AdvancedSearch = (props: Props) => {
+  const classes = useStyles();
+  const { author, onAuthorChange } = useSearchValue();
+
+  const { openDialog } = props;
+
+  return (
+    <>
+      {openDialog && (
+        <div className={classes.root}>
+          <FormControl fullWidth>
+            {/* Author */}
+            <Grid container spacing={2}>
```

Comment:
The suggestion should make the `label` and the `input` on the the same line
![image](https://user-images.githubusercontent.com/58532267/158453295-b6689741-3c6f-4f02-9477-6a74b2025f4c.png)

```suggestion
            <Grid container justifyContent='center' spacing={2}>
```

Reply:
I just used MUI input instead 😋